modules/ROOT/pages/app-dashboards.adoc

```diff
@@ -95,87 +82,553 @@ The following charts are available only if you are using Unified Agent and have
+
+JVM - CPU utilization::
+The average percentage of CPU utilized by the JVM, grouped by worker in automatic intervals. The JVM used in this chart is the Mule runtime JVM process. The calculation for CPU utilization is relative to the physical server on which the Mule runtime is deployed. For example, in RTF, the resource usage here indicates the the JVM's utilization with respect to the entire server, and not with respect to the Pod maximum CPU allocation.
+
+JVM - Heap memory utilization::
+The average amount of heap memory utilized by the JVM, grouped by worker in automatic intervals. The JVM used in this chart is the Mule runtime JVM process.
+
+JVM - Thread count::
+The average number of threads of the JVM, grouped by worker in automatic intervals. The JVM used in this chart is the Mule runtime JVM process.
+
+[[inbound-charts]]
+== Inbound
+
+Inbound metrics are retained for 24 hours.
+The following charts appear on the *Inbound* dashboard:
+
+Inbound - Total Requests by Response Type::
+The sum of inbound requests, grouped by type of response in automatic intervals. Responses are OK or Failed. If a Mule app has more than one connector type, all types are included in this metric.
+
+Inbound - Average Response Time::
+The average response time of inbound requests, grouped by application in automatic intervals. If a Mule app has more than one connector type, all types are included in this metric.
+
+Inbound - Response Time by Endpoint::
+The average, maximum, and minimum and total response time, grouped by endpoint.
+* Average: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+* Maximum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 500ms; red if the response time is greater than or equal to 500ms.
+* Minimum: Green if the response time is less than 300ms; yellow if the response time is greater than 300ms and less than 600ms; red if the response time is greater than or equal to 600ms.
+
+Inbound - Total Requests by Endpoint::
+The sum of inbound requests in the period, grouped by endpoint.
+
+Response Time graphs (99, 90, 75, and 50 percentile inbound)::
+The average response time of the 99, 90, 75, and 50 percentiles of inbound requests, grouped by application in automatic intervals. The unit for these metrics is milliseconds (ms).
+
+Inbound - Maximum Response Time::
+The maximum response time, grouped by endpoint in automatic intervals.
+
+Inbound - Failure Total Request Count::
+The sum of failed inbound requests, grouped by type of response in automatic intervals.
+
+Slow Request Times::
+The average of response times that are largest than 1 second, grouped by endpoint in automatic intervals. The sum of total calls in which the response time is greater than 1000 ms.
```

Comment:
The sum of inbound requests whose response time is greater than 1 second, grouped by ...

Reply:
```suggestion
The sum of inbound requests whose response time is greater than 1 second, grouped by endpoint in automatic intervals. The sum of total calls in which the response time is greater than 1000 ms.
```